testing/oak_echo_raw_bin/src/asm/boot.s

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
@andrisaar could this be encapsulated in an existing crate or a new one that will be shared across binaries?